sabnzbd/skintext.py

```diff
@@ -644,6 +644,10 @@
     "ssl_verify-strict": TT("Strict"),
     "srv-priority": TT("Priority"),  #: Server priority
     "explain-svrprio": TT("0 is highest priority, 100 is the lowest priority"),  #: Explain server priority
+    "srv-required": TT("Required"),  #: Server required tickbox
+    "explain-required": TT(
+        "Download will be paused while connection or login fails for this server"
```

Comment:
I don't think this explains the actual way it works:
`In case of connection failures, the download queue will be paused for a few minutes instead of skipping this server`. Or something like that.